src/modules/Store.js

```diff
@@ -0,0 +1,33 @@
+export default class Store {
+  constructor() {
+    this.task_data = JSON.parse(localStorage.getItem('tasks_data'))
+      || [];
+  }
+
+  updateLocalStorage() {
+    localStorage.setItem('tasks_data', JSON.stringify(this.task_data));
+  }
+
+  add(obj) {
+    this.task_data.push(obj);
+    this.updateLocalStorage();
+  }
+
+  remove(index) {
+    this.task_data.splice(index, 1);
+    this.task_data = this.task_data.filter((obj, index) => {
+      obj.index = index;
+      return true;
+    });
+    this.updateLocalStorage();
+  }
+
+  updateDescription(index, description) {
+    this.task_data[index].description = description;
```

Comment:
- [ ] Items can be added to the list correctly! 👌 However, _Index_ property of added item does not match new array length. For instance, if a 5th item is added to the list, the index of that item should equal to 5. Let's improve our code so that it meets this requirement! 😃 

**Mismatch in index property and array length:**
![image](https://github.com/Benawi/Microverse-To-Do-list/assets/12745474/a9fd9a9c-ed0a-42b0-b5b8-bcd97dbe27c5)